modules/neovim/hm.nix

```diff
@@ -5,6 +5,10 @@
   ...
 }:
 {
+  imports = [
+    ./nixvim.nix
+    ./nvf.nix
+  ];
   options.stylix.targets.neovim = {
```

Comment:
the target names remain neovim, nvf, nixvim. shouldnt they be changed to eg neovim.nvf, neovim.nixvim, neovim.vanilla to reflect that they are part of the neovim module now? is there an official way of defining submodules in stylix?